README.md

```diff
@@ -197,75 +197,144 @@ Alternatively, you can also find your Subscription ID in any subscription notifi
 
 As noted above, we currently support only the `id-only` payload type.  This means that events will contain a reference to the focus resource related to the event, but you will need to call back to a FHIR R4 or proprietary athenahealth API endpoint if you want to retrieve the latest content for that resource.  While this introduces an extra step, it also reduces the risk of accidental PHI exposure by keeping all access control at the athenahealth API layer.  It also helps avoid some ordering-related gotchas (see also [Event Ordering](#event-ordering) below).
 
-In addition to the focus resource ID, some events MAY also contain additional context such as the related Patient ID and/or Department ID (see example below).
-
-As per the Subscriptions R5 Backport IG, the first entry in each notification Bundle will always be a SubscriptionStatus resource which contains metadata about the Subscription associated with the event as well as metadata about the event(s) contained in the Bundle (see `entry[0].resource.notificationEvent[*].focus` below).  At present this will be the only entry in the Bundle since `full-resource` payload is not supported at this time.
+As per the Subscriptions R5 Backport IG, the first entry in each notification Bundle will always be a SubscriptionStatus resource which contains metadata about the Subscription associated with the event as well as metadata about the event(s) contained in the Bundle (see `entry[0].resource.notificationEvent[*].focus` below).  The second entry will be `AuditEvent` resource which will have information about the event. Additional Detail of the event would be part of the `extension[]` array. Note that extensions would be different for different events. 
```

Comment:
@prasannakumarramani we should also update the example payloads in the sample code (see https://github.com/athenahealth/aone-fhir-subscriptions/blob/main/samplecode/java/send-appointment-notification.sh for example) to include the new payload structure, i.e. `extensions` instead of `additionalContext`.